buildenv/jenkins/variables/defaults.yml

```diff
@@ -57,6 +57,9 @@ artifactory:
   numArtifacts:
     osu: 30
     unb: 5
```

Comment:
Why set it for `unb` since `useBuildDiscarderNumArtifacts` is set to `false` and uses `build_discarder.logs`?